bears/js/JSONFormatBear.py

```diff
@@ -1,23 +1,16 @@
 import json
 from collections import OrderedDict
 
+from coala_utils.param_conversion import negate
```

Comment:
please put this import-reorder into an own commit, it doesn't belong to this one :+1: